src/ResourceManagement/Automation/AutomationManagement/Microsoft.Azure.Management.Automation.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.Automation
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.Automation">
-      <PackageVersion>2.0.2</PackageVersion>
+      <PackageVersion>2.0.4</PackageVersion>
```

Comment:
@shahabhijeet  

We see 2.0.3 is already released to nuget. AFAIK we did not release this (Also from Azure repo history 2.0.2 was the last version). 

Can we track down how 2.0.3 was released and by who? 

https://www.nuget.org/packages/Microsoft.Azure.Management.Automation/